### ISSUE TITLE
Refile jumbles notes in target notebook

### ISSUE THREAD

**Issue by velardevines** [orgzly/orgzly-android#524]
- [x] I have searched for existing issues that may be the same as or related to mine.

Notes in the target notebook are jumbled after using Refile to move a note containing child notes.  

It's difficult to explain the target notebook structure after Refile because notes are jumbled throughout other notes which already existed in the target notebook.  

I'm not a programmer however if you explain what information would be helpful, I'll offer as much as I can.

Thanks. Orgzly is great application; using it every day.

--Bill

**Comment by nevenz** [comment]
It would be helpful if you can reproduce it in a small notebook (perhaps `Getting Started with Orgzly`) and list the steps taken. What app version is this?

**Comment by velardevines** [comment]
Ok, I'll try to reproduce the problem. I'm running version 1.7.3

--Bill

**Comment by velardevines** [comment]
I've spent a few minutes using Getting Started Notebook but cannot
reproduce the problem. I think close this issue. If it happens again I'll
record the structure of the target notebook.

I have made some app setting changes since the initial problem. All
notebooks are now on local sd card storage. Before there was a combination
of local storage and Dropbox notebooks. I've also enabled auto-sync.

Thanks for replying. Orgzly is used to manage Todos for all my projects;
it's used every day.

--Bill
